torch/utils/_triton.py

```diff
@@ -49,9 +50,8 @@ def has_triton_tma_device():
             and not torch.version.hip
         ):
             try:
-                from triton.language.extra.cuda import (  # noqa: F401
-                    experimental_device_tensormap_create1d,
-                    experimental_device_tensormap_create2d,
+                from triton.language import (  # noqa: F401
+                    _experimental_make_tensor_descriptor,
```

Comment:
This change causes has_triton_tma_device() to always return false w/triton 3.2, because triton 3.2 doesn't have _experimental_make_tensor_descriptor.

Reply:
Reverted - this change is not needed any more in this PR.